ci/jenkins/test.sh

```diff
@@ -184,7 +184,7 @@ function clean_antrea {
     for antrea_yml in ${WORKDIR}/*.yml; do
         kubectl delete -f $antrea_yml --ignore-not-found=true || true
     done
-    docker images | grep 'antrea' | awk '{print $3}' | xargs -r docker rmi || true
+    docker images --format "{{.Repository}}:{{.Tag}}" | grep 'antrea'| xargs -r docker rmi || true
     docker images | grep '<none>' | awk '{print $3}' | xargs -r docker rmi || true
```

Comment:
Not related to this change, however better to take care of "none" tagged image clean up on next line as well, it also deletes images based  based on image id, rather than tag